### PR TITLE
feat(testing): add unit tests for common/util

### PR DESCRIPTION
**Pull request by jsjoeio**
This PR adds several unit tests for the `src/common/util` functions.

## Screenshot

image

It increases our code coverage (Lines) from 46.88% to 49.53% (↑ 2.65%)
🟩 100% coverage for `src/common/util`
image

## TODOS
- [x] write tests for `logError`
- [x] write tests for `getOptions`
- [x] write tests for `generateUuid`
- [x] update screenshot and percentage

### CODE REVIEW

File: test/package.json [hunk 1]
```diff
@@ -1,11 +1,12 @@
 {
-  "#": "We must put jest in a sub-directory otherwise VS Code somehow picks up",
```
Comment (oxy) [inline test/package.json:2]: Half the comment went missing! I suspect this may be because both of them used the `"#"` key.

**Review by oxy — CHANGES_REQUESTED** [review]
PR looks good! Might have to combine the two comments in `test/package.json` into a single line to avoid future issues, though.

File: test/package.json (hunk 1 above)
Comment (jsjoeio) [reply test/package.json:2]: Weird, good catch!

**Review by oxy — APPROVED** [review]
LGTM!

**Pull request merged**